pkg/server/status.go

```diff
@@ -41,14 +42,16 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 		Type:   runtime.NetworkReady,
 		Status: true,
 	}
+
+	// Load the latest cni configuration to be in sync with the latest network configuration
+	if err := c.netPlugin.Load(cni.WithLoNetwork(), cni.WithDefaultConf()); err != nil {
+		logrus.WithError(err).Errorf("Failed to load cni configuration")
+	}
```

Comment:
might want to use an else if in this case as we don't care much about status() being successful for a prior load if we can't load properly with the current config.

If so we can cp the networkCondition assignments below to the if case and modify the else case to indicate which failure case we are in.. load or get status.